cmd/cassandra-writer/main.go

```diff
@@ -36,7 +37,8 @@ type config struct {
 	LogLevel   string `env:"MF_CASSANDRA_WRITER_LOG_LEVEL"     envDefault:"info"`
 	ConfigPath string `env:"MF_CASSANDRA_WRITER_CONFIG_PATH"   envDefault:"/config.toml"`
 	BrokerURL  string `env:"MF_BROKER_URL"                     envDefault:"nats://localhost:4222"`
-	JaegerURL  string `env:"MF_JAEGER_URL"                     envDefault:"localhost:6831"`
+	JaegerURL  string `env:"MF_JAEGER_URL"                 envDefault:"localhost:6831"`
+	InstanceID string `env:"MF_CASSANDRA_WRITER_INSTANCE_ID"   envDefault:""`
```

Comment:
envDefault of JaegerURL is not algined with others

```suggestion
	JaegerURL  string `env:"MF_JAEGER_URL"                     envDefault:"localhost:6831"`
	InstanceID string `env:"MF_CASSANDRA_WRITER_INSTANCE_ID"   envDefault:""`
```